ss-tproxy.conf

```diff
@@ -71,6 +71,7 @@ ipts_proxy_dst_port='1:65535'   # 黑名单 IP 的哪些端口走代理，多个
 opts_ss_netstat='auto'                  # auto/ss/netstat，用哪个端口检测工具，见 README
 opts_ping_cmd_to_use='auto'             # auto/standalone/parameter，ping 相关，见 README
 opts_hostname_resolver='auto'           # auto/dig/getent/ping，用哪个解析工具，见 README
+opts_ignore_resolv='false'              # true/false, 定义是否修改 resolv.conf，见 README
```

Comment:
I think you can add the option value `ignore` in` opts_overwirte_resolv` to achieve the same purpose.